lib/web/resources_test.go

```diff
@@ -429,6 +435,311 @@ func TestListResources(t *testing.T) {
 	}
 }
 
+func TestListResourcesPing(t *testing.T) {
```

Comment:
Would listing resources in various versions be a better way to cover this functionality, without the need to assert the number of ping calls?

Reply:
actually... i'm not sure why i felt like i needed to check it here, i removed it, and started the test with positive